scripts/run_tests.sh

```diff
@@ -20,7 +20,7 @@ function display_result {
   fi
 }
 
-make test-requirements
+poetry lock --check
```

Comment:
Keeping the `Makefile` entry would prevent much changes such as these, as this is used as an abstraction for most targets.

Reply:
Agreed, I'll restore these make targets.

Reply:
Restored here, as well as in `DDL-API`, and `admin`.